### PR TITLE
podofo: 0.9.6 -> 0.9.7

### PR DESCRIPTION
**Pull request by ajs124**
###### Motivation for this change
#90923 cc @redvers @ckauhaus 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rmcgibbo** [comment]
Here's the build report from my bot, which appeared to crash when posting for some reason related to the github API:

------------------
Result of `nixpkgs-review pr 112548` at 08c00e8e run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/8009bed4e9bc4006fb59d3bec5c66e4e">scribusUnstable</a></li></li>
  </ul>
</details>
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/402972f6b90b0c6ab2c4dd83749b6741">calibre</a></li></li>
    <li><a href="https://gist.github.com/abbf17fcd7a08ace74c4277e0294bb15">gImageReader</a></li></li>
    <li><a href="https://gist.github.com/448ed5087689dca412dad14e4cf01ca6">horizon-eda</a></li></li>
    <li><a href="https://gist.github.com/0303c4ba56b61cc0fcb9fd33032cabd8">krename</a></li></li>
    <li><a href="https://gist.github.com/d7f9f7aa05e392d64b2416013154ed3e">offrss</a></li></li>
    <li><a href="https://gist.github.com/1bfb6d6e277aaa5bfbff320c443ba91e">podofo</a></li></li>
    <li><a href="https://gist.github.com/f9d355e84b44f4264952f64e5c2f078a">scribus</a></li></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>
Package does not have a maintainer. Consider adding yourself?
Near pkgs/development/libraries/podofo/default.nix:31:3:

```
   |
31 |   meta = with lib; {
   |   ^
```

</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>
`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.
Near pkgs/development/libraries/podofo/default.nix:35:5:

```
   |
35 |     license = with licenses; [ gpl2 lgpl2 ];
   |     ^
```

</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>
`lgpl2` is a deprecated license, check if project uses `lgpl2Plus` or `lgpl2Only` and change `meta.license` accordingly.
Near pkgs/development/libraries/podofo/default.nix:35:5:

```
   |
35 |     license = with licenses; [ gpl2 lgpl2 ];
   |     ^
```

</li>
  </ul>
</details>

File: pkgs/development/libraries/podofo/default.nix [hunk 1]
```diff
@@ -3,23 +3,14 @@
...
 
   src = fetchurl {
     url = "mirror://sourceforge/podofo/${pname}-${version}.tar.gz";
-    sha256 = "0wj0y4zcmj4q79wrn3vv3xq4bb0vhhxs8yifafwy9f2sjm83c5p9";
+    sha256 = "1f0yvkx6nf99fp741w2y706d8bs9824x1z2gqm3rdy5fv8bfgwkw";
   };
 
-  patches = [
-    # https://sourceforge.net/p/podofo/tickets/24/
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/podofo/default.nix:16]: Please remove fetchpatch from inputs.

File: pkgs/applications/office/scribus/unstable.nix [hunk 2]
```diff
@@ -43,6 +43,13 @@ mkDerivation rec {
     sha256 = "sha256-1CV2lVOc+kDerYq9rwTFHjTU10vK1aLJNNCObp1Dt6s=";
   };
 
+  patches = [
```
Comment (ajs124) [inline pkgs/applications/office/scribus/unstable.nix:46]: cc maintainers @erictapen @Kiwi

Comment (erictapen) [reply pkgs/applications/office/scribus/unstable.nix:46]: Thanks for pinging, the fix LGTM!

**Pull request merged**